tee-worker/local-setup/launch.py

```diff
@@ -59,11 +59,12 @@ def setup_worker(work_dir: str, source_dir: str, std_err: Union[None, int, IO],
 
 
 def run_worker(config, i: int, log_config_path):
-    log = open(f"{log_dir}/worker{i}.log", "w+")
+    id = config.get('id', i)
+    log = open(f"{log_dir}/worker-{id}.log", "w+")
     # TODO: either hard-code 'local-setup' directory, or take from input config.json
-    w = setup_worker(f"tmp/w{i}", config["source"], log, log_config_path)
+    w = setup_worker(f"tmp/w-{id}", config["source"], log, log_config_path)
```

Comment:
Hmm is it backwards compatible? We had like `w0`, `w1`...

Can we make it `w-{id}`, and if not present,  `w0`, `w1`...? 

It might affect `deploy.sh`

Reply:
I cannot find usages of `launch.py` in other scripts, can you guide me.
If something relies on old naming, this might break it.

We can use old convention if `id` is not present.

Reply:
I don't think this should affect `deploy.sh` because in it, we are using `opt/worker/` for storing and not using `tmp` in the repo and not using  `launch.py`. This should only affect local development setup, but I will try setting up multiple workers using `deploy.sh` .

Reply:
I see - yes it only affects local dev